test_integration/lib/provisioning.dart

```diff
@@ -94,36 +81,9 @@ Future<Map<String, dynamic>> getTokenRequest() async {
   final r = await const RetryOptions(
     maxAttempts: 5,
     delayFactor: Duration(seconds: 2),
-  ).retry(() => http.get(Uri.parse(authURL)));
+  ).retry(() => http.get(Uri.parse(_authURL)));
   print('tokenRequest from tokenRequest server: ${r.body}');
   return Map.castFrom<dynamic, dynamic, String, dynamic>(
     jsonDecode(r.body) as Map,
   );
 }
-
-Future<Map<String, dynamic>> getTokenDetails(
```

Comment:
We may need this in `test_integration` when we write integration tests covering `authCallback`

Reply:
Thanks for catching that. I have put it back in with a8beb04fc7c83f66c07e1fccf255b3276586e53b